lib/collection/src/collection/point_ops.rs

```diff
@@ -103,7 +103,11 @@ impl Collection {
             let shard_requests = shard_to_op
                 .into_iter()
                 .map(move |(replica_set, operation)| {
-                    replica_set.update_with_consistency(operation, wait, ordering)
+                    replica_set.update_with_consistency(
+                        operation.into(), // TODO: Assign tag!
```

Comment:
I would prefer to use explicit conversion `WithMeta::from(operation)`, as in this case navigation and code usage search is much simpler. (yes, even with rust_analyser)